.gitignore

```diff
@@ -1,3 +1,5 @@
 /target
 /vidyut-cheda/data/
 /vidyut-cheda/dcs-data/
+/data/
```

Comment:
these have been moved into `vidyut-cheda` now, so I don't think the new ignores are needed